timApp/user/user.py

```diff
@@ -1284,6 +1284,14 @@ def grant_access(
             require_confirm=require_confirm,
         )
 
+    def remove_perm(self, block: ItemOrBlock, access_type: AccessType):
```

Comment:
Tässä voisi käyttää sanaa `access` eikä `perm`, koska tässä moduulissa käytetään jo tuota sanaa.

Reply:
Tässä kävi niin, että `user.py`ssa oli jo olemassa funktio `remove_access`, mutta jostain syystä siinä oltiin käytetty AccessTypen välittämiseen string-tyyppistä parametria tuon AccessType-enumin sijaan. En halunnut rikkoa mitään, mutten myöskään tehdä asiaa hankalasti, joten tein tuosta `remove_access`sta uuden version joka parametrisoidaan suoraan AccessTypella.

Reply:
Onnistuisiko sitten vaikka niin, että vaihtaa sen alkuperäisen `remove_access` muotoon:

```py
def remove_access(self, block_id: int, access_type: str | AccessType):
        if isinstance(access_type, AccessType):
            access_type = access_type.value
```

Näin pitäisi onnistua sallimaan kummatkin muodot yhdessä funktiossa.

Reply:
Joo, toimiihan tuo. Toki sekin olisi vaihtoehto että refaktotoi tuon aikaisemman remove_accessin tuohon mun versioon ja muuttaa viitteet muualla. Teettää toki enemmän työtä, mutta tekisi mielestäni koodista myös luettavampaa.

Reply:
Tämä muutettu ehdotuksen mukaiseksi, eipähän tätäkään enää käytetty muuta kuin testeissä joten varmaan turhaa uhrata tälle enempää aikaa.